packages/api/schema/stryker-core.json

```diff
@@ -253,6 +262,452 @@
           "default": true
         }
       }
+    },
+    "MutationSpecification": {
+      "anyOf": [
+        { "$ref": "#/definitions/MutationLevelName"},
+        { "$ref": "#/definitions/MutatorDefinition"},
+        { "$ref": "#/definitions/MutatorGroupName"}
+      ]
+    },
+    "MutationLevelName": { 
+      "title": "MutationLevelName", 
+      "type": "string"
```

Comment:
TODO: decide on the actual names

Reply:
We have also updated the supported-mutators documentation [here](https://github.com/Luctia/isep-mutation-testing-elements/blob/master/docs/supported-mutators.md), based on your [PR ](https://github.com/stryker-mutator/mutation-testing-elements/pull/1458). 

We also noticed a few mutation operators that were not in the documentation as well, so we tried to come up with new names for those following your convention.